src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private String name = "Isosceles Trapezoid";
+    private String color;
+    private double firstLeg, secondLeg, thirdLeg, height, smallPiece;
```

Comment:
Think about which variables should be local in the method and which should be class-level. The variable 'smallPiece' seems to be used only within the 'findArea' method, so it should be a local variable instead of a class field.